### PR TITLE
Update polish translation

### PR DESCRIPTION
**Pull request by ghost**
## Please check that your pull request pass this checks:

-  [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
-  [x] My code is both Python 2 and 3 compatible
-  [x] My code follows the code style of this project (PEP8, long lines allowed if make sense)
-  [x] I made sure there wasn't another Pull Request opened for the same update/change
-  [x] I have successfully tested my changes locally

## Types of changes

-  [ ] Bug fix (non-breaking change which fixes an issue)
-  [ ] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to change)
-  [ ] Adding a new region/hbo go version

## Description:
Just update and some corrections of polish translation.

### CODE REVIEW

**Comment by barqqlsky** [comment]
Just update of polish translate.

**Comment by arvvoid** [comment]
Hello @barqqlsky , thank you for submitting the translation. I can merge it as it is but before i do take a quick look at this strings that are still marked a fuzzy (meaning needing review).

```
#, fuzzy
msgctxt "#30001"
msgid "ERROR"
msgstr "BŁĄD"

#, fuzzy
msgctxt "#30002"
msgid "Setup failed"
msgstr "Instalacja nie powiodła się"

#, fuzzy
msgctxt "#30003"
msgid "Unsupported region"
msgstr "Nieobsługiwany region"

#, fuzzy
msgctxt "#30004"
msgid "An unexpected error has occurred."
msgstr "Wystąpił nieoczekiwany błąd."
```

If those are ok, please remove the fuzzy mark from them and update the pull request so I merge a completely clean translation.

File: resources/language/resource.language.pl_pl/strings.po [hunk 1]
```diff
@@ -422,43 +418,42 @@ msgstr ""
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 2]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30696"
 msgid "Home Category"
-msgstr "Strona główna Kategoria"
+msgstr "Strona główna"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 3]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30697"
 msgid "Group Home lists in special category"
-msgstr "Lista główna grupy w kategorii specjalnej"
+msgstr "Strona główna wyświetlana w kategorii specjalnej"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 4]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30800"
 msgid "Sync play time with HBO Go"
-msgstr "Synchronizacja czasu gry z HBO go"
+msgstr "Synchronizacja oglądania z HBO go"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 5]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30801"
 msgid "Send elapsed time to HBO Go"
-msgstr "Wyślij upływ czasu do HBO go"
+msgstr "Wyślij statusy obejrzane do HBO go"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 6]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30802"
 msgid "Get elapsed time from HBO Go"
-msgstr "Pobierz upływ czasu z HBO go"
+msgstr "Pobierz statusy obejrzane z HBO go"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 7]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30803"
 msgid "Set as watched on HBO Go"
-msgstr "Ustaw jako oglądany na HBO go"
+msgstr "Oznacz jako obejrzany"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

File: resources/language/resource.language.pl_pl/strings.po [hunk 8]
```diff
@@ -422,43 +418,42 @@ msgstr ""
...
 msgctxt "#30804"
 msgid "Set as unwatched on HBO Go"
-msgstr "Ustaw jako nieoglądany na HBO go"
+msgstr "Oznacz jako nieobejrzany"
 
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.pl_pl/strings.po]: Remove fuzzy if no longer necessary and the string is ok

**Review by arvvoid — CHANGES_REQUESTED** [review]
Please review the strings marked with #fuzzy and remove the fuzzy mark if they are ok.

**Comment by ghost** [comment]
I've checked it and everything is fine, so I deleted #fuzzy

**Comment by arvvoid** [comment]
Hello tnx, but It looks like there are some aditional problems:

TESTING resource.language.pl_pl:
```
./resources/language/resource.language.pl_pl/strings.po:309: this message is untranslated

msgctxt "#30733"
msgid "Home Lists"
msgstr ""

./resources/language/resource.language.pl_pl/strings.po:361: this message is untranslated

msgctxt "#30672"
msgid "Use generic video media type in listings"
msgstr ""

./resources/language/resource.language.pl_pl/strings.po:398: this message needs to be reviewed by the translator

#, fuzzy
#| msgctxt "#30693"
#| msgid "Include full details/RAW retrived data in Debug log"
msgctxt "#30693"
msgid "Include full details/RAW retrieved data in Debug log"
msgstr "Include full details/RAW retrived data in Debug log"

(the comments and fuzzy need to be removed)

./resources/language/resource.language.pl_pl/strings.po:409: this message needs to be reviewed by the translator

#, fuzzy
#| msgctxt "#30694"
#| msgid ""
#| "Cryptodome dependency not present on your system, HBO go add-on can't "
#| "run. For help consult install instructions/official support thread"
msgctxt "#30694"
msgid ""
"Cryptodome dependency not present on your system, HBO Go add-on can't run. "
"For help consult install instructions/official support thread"
msgstr ""
"Cryptodome dependency not present on your system, HBO go add-on can't run. "
"For help consult install instructions/official support thread"

(the comments and fuzzy need to be removed)

./resources/language/resource.language.pl_pl/strings.po:447: this message is untranslated

msgctxt "#30806"
msgid "Cache"
msgstr ""


./resources/language/resource.language.pl_pl/strings.po:451: this message is untranslated

msgctxt "#30807"
msgid "Use request cache"
msgstr ""

./resources/language/resource.language.pl_pl/strings.po:455: this message is untranslated

msgctxt "#30808"
msgid "Clear request cache"
msgstr ""

./resources/language/resource.language.pl_pl/strings.po:459: this message is untranslated

msgctxt "#30809"
msgid "Encrypt credentials"
msgstr ""

./resources/language/resource.language.pl_pl/strings.po:475: this message is untranslated

msgctxt "#30810"
msgid ""
"If you don't know why the option is disabled don't input credentials and "
"exit now!\n"
"\n"
"If you disabled the option on purpose and know exactly why then please "
"continue.\n"
"\n"
"If this is not intended for debugging/testing or because you experience "
"problems with lost credentials on the device please do NOT continue.\n"
"In case this is not intended, enable the credentials encryption option and "
"simply try again.\n"
"\n"
"This message will appear only now on credential input."
msgstr ""

```

**Review by arvvoid — APPROVED** [review]

**Comment by ghost** [comment]
ok, I translated the missing lines, but  I am not sure if I translated the following strings well, because I do not know the context:

msgctxt "#30733"
msgid "Home Lists"
 It,s a cattegory or what? I cannot find this in addon

msgctxt "#30672"
msgid "Use generic video media type in listings"
 I don't see any difference in media lists after swish option on/off

msgctxt "#30697"
msgid "Group Home lists in special category"
 What special category? I don't see any difference with on and off this option

**Comment by arvvoid** [comment]
Hello, if you decide to change those you can send the corrections in a new pull request.

"Group Home lists in special category" option create a label (named: "Home Lists") in the Home screen, when accessed shows all the usual categories that are shown directly on the home screen. After activating the option you might need to load a different add-on or exit Kodi and then go back to hbo to invalidate the cache to see the difference sometimes.

"Use generic video media type in listings" set the media type on directories always to generic video (usually this is not necessary but works better with some skins). 
The default behavior with the option off is to analyze the dir content and decide if set the dir media type to movie or tv show or episodes based on the majority of content in it. Visually on some skins makes little difference on other skins the difference is more pronounced if they have distinct views for movies and tv shows, episodes and videos. There are always subtle differences like view modes available, default view mode selected,  labels on the screen depending on the set content type.

@all-contributors please add @barqqlsky for translation

**Comment by arvvoid** [comment]
@all-contributors please add @barqqlsky for translations

**Pull request merged**